core/base/src/TSystem.cxx

```diff
@@ -55,6 +47,18 @@ allows a simple partial implementation for new OS'es.
 #include "RConfigure.h"
 #include "THashList.h"
 
+#ifdef WIN32
+#include <io.h>
+#endif
+#include <stdlib.h>
+#include <errno.h>
+#include <algorithm>
```

Comment:
Are all these headers really needed? I'd check `io.h`, `stdlib.h`, `errno.h`, `algorithm`, `sys/stat.h`.

Reply:
Only `sys/stat.h` stayed I've removed the rest.